### DISCUSSION TITLE
Can't track with Langfuse when integrated with API on Telegram or Discord or Email.

### DISCUSSION THREAD

**Discussion by adolfousier** [FlowiseAI/Flowise#2053] [General]
Hi everyone.

I just noticed that we can't track with Langfuse when integrated through API on Telegram or Discord or Email. 

Anyway to fix this? Thanks

**Comment by adolfousier** [comment]
Is it normal? Do I have to track at code level? Thank you all in advance.

**Reply by HenryHengZJ** [reply]
have you turned on the langfuse analytic?
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/0c2d49c7-e8cb-4ea5-b6e3-27616fb2adea)
